core/src/main/java/bio/terra/pearl/core/service/workflow/EnrollmentService.java

```diff
@@ -83,8 +97,8 @@ public PreEnrollmentResponse createAnonymousPreEnroll(
     }
 
     @Transactional
-    public HubResponse<Enrollee> enroll(ParticipantUser user, PortalParticipantUser ppUser, EnvironmentName envName,
-                                            String studyShortcode, UUID preEnrollResponseId) {
+    public HubResponse<Enrollee> enroll(String portalShortcode, ParticipantUser user, PortalParticipantUser ppUser, EnvironmentName envName,
+                                            String studyShortcode, UUID preEnrollResponseId, boolean isProxy) {
```

Comment:
Let's reshuffle these arguments so that the portal/study/env args are together.  
```suggestion
    public HubResponse<Enrollee> enroll(String portalShortcode, EnvironmentName envName, String studyShortcode, ParticipantUser user, PortalParticipantUser ppUser, UUID preEnrollResponseId, boolean isProxy) {
```